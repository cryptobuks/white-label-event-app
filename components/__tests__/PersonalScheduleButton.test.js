import React from 'react';
import renderer from 'react-test-renderer';
import PersonalScheduleButton from '../PersonalScheduleButton';

describe('PersonalScheduleButton', () => {
  it('renders correctly', () => {
    const comp = <PersonalScheduleButton />;

    const tree = renderer.create(comp).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
