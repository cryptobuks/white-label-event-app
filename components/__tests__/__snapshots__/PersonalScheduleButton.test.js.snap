// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonalScheduleButton renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hitSlop={undefined}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#35e2aa",
        "bottom": 0,
        "flex": 1,
        "height": 56,
        "justifyContent": "center",
        "position": "absolute",
        "shadowColor": "black",
        "shadowOffset": Object {
          "x": -5,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 12,
        "width": 750,
      },
      null,
      null,
    ]
  }
  testID={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "white",
        "fontSize": 16,
        "fontWeight": "600",
      }
    }
  >
    My Schedule
  </Text>
</View>
`;
